packages/dom/test/unit/lib/fixtures/nestedElementsTestComponent.tsx

```diff
@@ -0,0 +1,20 @@
+import { ReactElement } from "react";
+
+export function NestedElementsTestComponent(): ReactElement {
+  return (
+    <span data-testid="grandparent">
+      <span data-testid="parent">
+        <span data-testid="child">
+          <span>
+              <span>
+                  <span>
+                      <span data-testid="deep-child"></span>
+                  </span>
+              </span>
+          </span>
```

Comment:
2-spaces for indentation 🙂 
```suggestion
          <span>
            <span>
              <span>
                <span data-testid="deep-child"></span>
              </span>
            </span>
          </span>
```

Reply:
Thanks @JoseLion, I have addressed this [here](https://github.com/stackbuilders/assertive-ts/pull/132/commits/229b78ed9e43e637f0b4c8f76929f913d611ae16).